src/components/Dropdown/Dropdown.tsx

```diff
@@ -218,6 +220,9 @@ function useDropdownContextInstance(
   const toggleDropdown = (willBeOpen: boolean) => {
     if (!willBeOpen) {
       setHasTyped(false);
+      if (index === -1) {
+        updateFocusedItem(0);
```

Comment:
these updates would change the behavior for the `SearchBar` dropdown too, right, which we don't want?

Reply:
hmm you're right, do you think making a new prop for `Dropdown` on whether or not it's `FilterSearch` would make sense? or should I try modifying `FilterSearch`'s `onToggle` method

Reply:
I think it's best if we avoid adding new props, especially if its usage is to going to be for specific component implementation only. If we can update onToggle method for this behavior, that would be great!